package.json

```diff
@@ -62,85 +62,92 @@
-        "@angular/animations": "~10.2.3",
-        "@angular/cdk": "~10.2.7",
-        "@angular/common": "~10.2.3",
-        "@angular/compiler": "~10.2.3",
-        "@angular/core": "~10.2.3",
-        "@angular/flex-layout": "~10.0.0-beta.32",
-        "@angular/forms": "~10.2.3",
-        "@angular/material": "~10.2.7",
-        "@angular/material-moment-adapter": "~10.2.7",
-        "@angular/platform-browser": "~10.2.3",
-        "@angular/platform-browser-dynamic": "~10.2.3",
-        "@angular/router": "~10.2.3",
-        "@covalent/core": "~3.1.0",
-        "@covalent/highlight": "~3.1.0",
-        "@covalent/markdown": "~3.1.0",
-        "@covalent/text-editor": "~3.1.0",
+        "@angular-material-components/datetime-picker": "~7.0.1",
+        "@angular-material-components/moment-adapter": "~7.0.0",
+        "@angular/animations": "~13.3.1",
+        "@angular/cdk": "~13.3.1",
+        "@angular/common": "~13.3.1",
+        "@angular/compiler": "~13.3.1",
+        "@angular/core": "~13.3.1",
+        "@angular/flex-layout": "~13.0.0-beta.38",
+        "@angular/forms": "~13.3.1",
+        "@angular/material": "~13.3.1",
+        "@angular/material-moment-adapter": "~13.3.1",
+        "@angular/platform-browser": "~13.3.1",
+        "@angular/platform-browser-dynamic": "~13.3.1",
+        "@angular/router": "~13.3.1",
+        "@covalent/markdown": "~4.1.0",
         "@ngx-translate/core": "~13.0.0",
         "@ngx-translate/http-loader": "~6.0.0",
-        "@schematics/angular": "~10.2.0",
-        "@swimlane/ngx-charts": "~16.0.0",
-        "@types/mousetrap": "1.6.3",
+        "@schematics/angular": "~13.3.0",
+        "@swimlane/ngx-charts": "~20.1.0",
+        "@types/mousetrap": "~1.6.9",
         "angular-resizable-element": "~3.3.0",
-        "angular-resize-event": "~2.0.1",
-        "angular2-hotkeys": "~2.2.0",
+        "angular-resize-event": "~3.1.1",
+        "angular2-hotkeys": "~2.4.0",
+        "easymde": "~2.16.1",
         "hammerjs": "~2.0.8",
         "json-schema-to-typescript": "~8.1.0",
         "jszip": "~3.2.2",
+        "marked": "~4.0.12",
         "moment": "~2.24.0",
+        "mousetrap": "~1.6.5",
         "natural-orderby": "~2.0.3",
-        "ngx-quill": "~12.0.1",
+        "ngx-quill": "~16.2.0",
         "palette-creator": "~0.5.4",
         "quill": "~1.3.7",
         "rxjs": "~6.5.4",
-        "tslib": "~1.11.1",
-        "zone.js": "~0.10.2"
+        "showdown": "^2.0.3",
+        "tslib": "^2.0.0",
+        "zone.js": "~0.11.4"
     },
     "devDependencies": {
-        "@angular-devkit/build-angular": "~0.1002.0",
-        "@angular-devkit/build-ng-packagr": "~0.1002.0",
-        "@angular-devkit/schematics-cli": "~0.1002.0",
-        "@angular/cli": "~10.2.0",
-        "@angular/compiler-cli": "~10.2.3",
-        "@angular/language-service": "~10.2.3",
+        "@angular-devkit/build-angular": "~13.3.1",
+        "@angular-devkit/schematics-cli": "~0.1200.0-next.8",
```

Comment:
shouldn't this be 0.1300 when we use Angular 13?

Reply:
its the neewst version